src/components/modes/teacher/TeacherView.js

```diff
@@ -97,26 +97,27 @@ export class TeacherView extends Component {
     dispatchGetUsers();
   }
 
-  handleDelete = async ({ id, uri }) => {
-    const {
-      dispatchDeleteAppInstanceResource,
-      dispatchDeleteFile,
-    } = this.props;
+  handleDelete = async ({ id, _id, uri }) => {
```

Comment:
From what I see below, this should not need to receive an `_id` so no need to handle that case. (See below).